crates/rbuilder/src/live_builder/order_input/txpool_fetcher.rs

```diff
@@ -24,11 +24,20 @@ pub async fn subscribe_to_txpool_with_blobs(
     results: mpsc::Sender<ReplaceableOrderPoolCommand>,
     global_cancel: CancellationToken,
 ) -> eyre::Result<JoinHandle<()>> {
-    let ipc_path = config
-        .ipc_path
-        .ok_or_else(|| eyre::eyre!("No IPC path configured"))?;
-    let ipc = IpcConnect::new(ipc_path);
-    let provider = ProviderBuilder::new().on_ipc(ipc).await?;
+    let mempool = config
+        .mempool_source
+        .ok_or_else(|| eyre::eyre!("No TX source configured"))?;
```

Comment:
No txpool source configured

Reply:
And if we fail on empty mempool - let's fail on config parsing stage if possible?

Reply:
> And if we fail on empty mempool - let's fail on config parsing stage if possible?

I have followed logic of already existing code. My understanding is that failing during config parsing would be undesirable because `rbuilder` still can receive orders (transacitons) via bundle API